nixos/doc/manual/configuration/user-mgmt.chapter.md

```diff
@@ -32,8 +32,7 @@ account will cease to exist. Also, imperative commands for managing users and
 groups, such as useradd, are no longer available. Passwords may still be
 assigned by setting the user\'s
 [hashedPassword](#opt-users.users._name_.hashedPassword) option. A
-hashed password can be generated using `mkpasswd -m
-  sha-512`.
+hashed password can be generated using `mkpasswd -m yescrypt`.
```

Comment:
What's the rationale for this change? sha-512 `$6$` is still allowed as `good_schemes`.
Also note that `yescrypt` is the type when calling `mkpassword` without `-m`.

Reply:
I'll reference https://fedoraproject.org/wiki/Changes/yescrypt_as_default_hashing_method_for_shadow, where the author of libxcrypt explains why sha256crypt & sha512crypt are not optimal.

I'm aware it is the default, the ordering comes from the order in https://github.com/besser82/libxcrypt/blob/v4.4.28/lib/hashes.conf#L41.

I'm fine with removing the `-m` parameter entirely, and I'm against sticking with sha512 explicitly, because documentation gets outdated far too quickly.